### ISSUE TITLE
Copy is not available in the context menu in Firefox

### ISSUE THREAD

**Issue by spadgett** [sourcelair/xterm.js#1003]
Copy is not available in the context menu in Firefox.

<img width="924" alt="screen shot 2017-09-20 at 7 55 06 am" src="https://user-images.githubusercontent.com/1167259/30642567-62419034-9dd9-11e7-885b-16c9dd0e6a01.png">

## Details
- Browser and browser version: Firefox 55.0.3 (64-bit)
- OS version: macOS Sierra 10.12.6
- xterm.js version: 2.9.2 (also tested current master)

### Steps to reproduce

1. Run the demo https://github.com/sourcelair/xterm.js#demo
2. Open http://0.0.0.0:3000/ in Firefox
3. Run a command like `ls`
4. Select some text
5. Right click

No copy is found in the context menu.

**Comment by parisk** [comment]
Hi @spadgett, thanks for reporting this.

Can you please let me know if this happens in the `v3` branch also?

**Comment by spadgett** [comment]
@parisk Works OK for me in the `v3` branch

**Comment by Tyriar** [comment]
Strange, I don't think anything changed around the textarea/context menu.

**Comment by Tyriar** [comment]
I guess I'll close this off, let us know if you see it again.

**Comment by Kalen66100** [comment]
I've tested it on Firefox ("windows") with V3 and no copy/paste options in the context menu.

**Comment by shanytc** [comment]
the main issue for right click -> Save Image.. context-menu in xtermjs, is because '.xterm-helpers' is having position:absolute. Try change it to position: relative.
When you right click on the canvas, xtermjs will calculate the x,y position of the mouse and move the <textarea/> to that position under the

element so that it will then capture the right click (copy & paste) context menu and also include keyboard support.
So basically sometimes the "xterm-helpers" is not positioned under the actual canvas and so you get "Save image..." context menu because that's the canvas's context menu and not the <textarea/> element's context menu.
